common.sh

```diff
@@ -3,6 +3,8 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 USER=`whoami`
 WORKING_DIR=${WORKING_DIR:-"/opt/dev-scripts"}
+NODES_FILE=${NODES_FILE:-"${WORKING_DIR}/ironic_nodes.json"}
+NODES_PLATFORM=${NODES_PLATFORM:-"libvirt"}
```

Comment:
This isn't working as I expected, the WORKING_DIR is /opt/dev-scripts for NODES_FILE even if I override WORKING_DIR, I'll have to add a seperate conditional

Reply:
Maybe I don't get what you meant but It seems to work for me..

Reply:
Nevermind, I see what happened in your env now.